packages/create-block/README.md

```diff
@@ -141,7 +141,7 @@ For example, running the `start` script from inside the generated folder (`npm s
 
 ## External Project Templates
 
-[Click here](https://github.com/WordPress/gutenberg/tree/HEAD/packages/create-block/docs/external-template.md) for information on External Project Templates
+[Click here](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block-tutorial-template/) for information on External Project Templates
```

Comment:
It should be a different link:

https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/packages-create-block-external-template/

Reply:
oh shooot will fix in next PR.